tests/userTest.js

```diff
@@ -451,7 +503,7 @@ describe('Main: Currently testing user management,', function () {
       data => {
         data.appendElement('users', '', users => {
           users.appendElement('element', '', config.adminUsername)
-            .appendElement('element', '', config.testUser)
+            .appendElement('element', '', MatchersV3.string(config.testUser))
```

Comment:
I don't think we need matchers here, the mock server and the provider should respond with the exact value